Examples/S3_AWSSDK/template.yaml

```diff
@@ -0,0 +1,43 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Transform: AWS::Serverless-2016-10-31
+Description: SAM Template for AWS SDK Example
+
+Resources:
+  # Lambda function
+  AWSSDKExample:
+    Type: AWS::Serverless::Function
+    Properties:
+      CodeUri: .build/plugins/AWSLambdaPackager/outputs/AWSLambdaPackager/AWSSDKExample/AWSSDKExample.zip
+      Timeout: 60
+      Handler: swift.bootstrap
+      Runtime: provided.al2
+      MemorySize: 512
+      Architectures:
+        - arm64
+      Environment:
+        Variables:
+          # by default, AWS Lambda runtime produces no log
+          # use `LOG_LEVEL: debug` for for lifecycle and event handling information
+          # use `LOG_LEVEL: trace` for detailed input event information
+          LOG_LEVEL: trace
```

Comment:
do we really want trace here? I think thats a bad example.

Reply:
OK, I'll add a note to verify that the TRACE level has been kept in the new runtime